docs/extensions/reference/_posts/1970-01-06-CLI.md

```diff
@@ -104,7 +104,7 @@ Extension initialized!
 After initialization, your extension folder structure will look as follows:
 
 ```
-Restaurants/
+{{ site.example.devName }}.{{ ste.example.extensionName }}/
```

Comment:
Please, explain that `{{ site.example.devName }}` is just a developer used as an example.